python/erdos/operator.py

```diff
@@ -386,7 +388,7 @@ def destroy(self):
         """
 
 
-class OperatorConfig:
+class OperatorConfig(Generic[T]):
```

Comment:
```suggestion
class OperatorConfig():
```

`OperatorConfig` does not have any generic types.